src/bptl/work_units/zgw/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+
+class DefaultService(models.Model):
+    """default ZGW services for particular tasks"""
```

Comment:
for future - can you please start docstrings with a capital letter? They are normal English sentences :)

Reply:
Sure!